examples/elliptical_rotations.py

```diff
@@ -0,0 +1,100 @@
+#  Visualize the acquisition of a movie where the rotation axis is not
+#  aligned with the image plane.
```

Comment:
The example doesn't run in a fresh environment as the `debug` directory doesn't exist. Maybe add a flag for saving, or a check if `debug` exists and if not create it.

Reply:
I realized later it doesn't work also because it is calling methods from tests...